sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ReceiverManager.cs

```diff
@@ -109,13 +113,32 @@ public virtual async Task ReceiveAndProcessMessagesAsync(CancellationToken cance
             }
         }
 
+        protected async Task ProcessOneMessageWithinScope(ServiceBusReceivedMessage message, string activityName, CancellationToken cancellationToken)
```

Comment:
Though they're not public, should the async members in this class conform to the `Async` suffix for consistency within the codebase?